File: python-venv/almost_even/kata_function/almost_even.py
def almost_even(total, parts):
    
    if parts == 1:
        return [total]
    if parts == total:
        return [1, 1]
    
    output = []
    if (total / parts).is_integer():
        for i in range(parts):
            output.append(int(total / parts))
        return output
    
    remainder = total % parts
    for j in range(parts):
        if remainder > 0:
            output.append(int(total / parts) + 1)
            remainder -= 1
        else:
            output.append(int(total / parts))

    return list(reversed(output))

#https://www.codewars.com/kata/529e2e1f16cb0fcccb000a6b/python

File: almost-even/almostEven.js
const almostEven = (total, parts) => {
    if (parts === 1) return [total];
    if (parts === total) return [1, 1];

    const output = [];
    if (total / parts === Math.floor(total / parts) === true) {
        for(let i = 0; i < parts; i++){ output.push(total / parts); };
        return output;
    };

    let remainder = total % parts;
    for(let j = 0; j < parts; j++){
        if(remainder > 0) {
            output.push(Math.floor(total / parts) + 1);
            remainder--;
        } else { output.push(Math.floor(total / parts)); };
    };

    return output.reverse();

};

module.exports = { almostEven };

//https://www.codewars.com/kata/529e2e1f16cb0fcccb000a6b/train/javascript

